G-game-on/Program.cs

```diff
@@ -0,0 +1,63 @@
+﻿// mandel
+
+Console.WriteLine("Welcome to NIM!");
+
+gameStart: ;
+Console.WriteLine("|||||||||||||||||||||||| (24)");  
+int matches = 24;
+bool playerTurn = true;
+
+drawPhase: ;
+
+if (playerTurn){
+    Console.WriteLine("How many matches do you want to draw?");
+        
+    if (!int.TryParse(Console.ReadLine(), out int userDraw)) {
+        Console.Write("Invalid input, try again!");
+        goto drawPhase;
+    }
+    
+    if (userDraw is > 3 or < 1) {
+        Console.Write("You can only draw 1,2 or 3 matches at time! ");
+        goto drawPhase;
+    }
+    if (matches < userDraw) {
+        Console.WriteLine("You can't draw more matches than there's left!");
+        goto drawPhase;
+    }
+    matches -= userDraw;
+}
+
+if (!playerTurn)
+{
+    int maxDraw = matches > 4 ? 4 : matches;
+    int aiDraw = Random.Shared.Next(1, maxDraw);
```

Comment:
The only thing I slightly dislike here is that if there's one match left, the AI tries to draw: `Random.Shared.Next(1, 1)` Which would mean between 1 inclusive and 1 exclusive. But it still works.